model_files/MRSSM2/FlexibleSUSY.m.in

```diff
@@ -96,6 +96,8 @@ ExtraSLHAOutputBlocks = {
          {24, FlexibleSUSYObservable`EDM[Fe[2]]},
          {25, FlexibleSUSYObservable`EDM[Fe[3]]},
          {26, FlexibleSUSYObservable`BrLToLGamma[Fe[2] -> {Fe[1], VP}]},
```

Comment:
I introduced (in [7ca8b16](https://github.com/FlexibleSUSY/FlexibleSUSY/pull/583/commits/7ca8b16d0d43a1ded7636321d80485e829b523b2)) a synonym ``BrLToLGamma[Fe, 2->1]`` that will exist along with ``BrLToLGamma[Fe@2-> {Fe@1, VP}]``.
We can introduce even more synonyms later.

Reply:
I agree that `BrLToLGamma[Fe, 2->1]` is much shorter. On the other hand, the initial syntax `BrLToLGamma[Fe[2] -> {Fe[1], VP}]` is very clear and unambiguous. In particular it makes absolutely clear that `2` and `1` are the generation indices of the fermion `Fe` and not something else (loop orders or "2 leptons in and 1 lepton out" or something like this). So, personally speaking I would prefer the old syntax `BrLToLGamma[Fe[2] -> {Fe[1], VP}]` to be supported as well.

Another point to consider: Is one of the syntax proposals (old or new) more-or-less consistent with decay specifications? For example one could imagine that in the future we may support certain specific decays with a syntax like `Decay[Fe[2] -> {Fe[2], hh[1]}]` or so. Such a syntax would then be very clear and consistent with the old syntax for `BrLToLGamma`.

Reply:
> I agree that `BrLToLGamma[Fe, 2->1]` is much shorter. On the other hand, the initial syntax `BrLToLGamma[Fe[2] -> {Fe[1], VP}]` is very clear and unambiguous. In particular it makes absolutely clear that `2` and `1` are the generation indices of the fermion `Fe` and not something else (loop orders or "2 leptons in and 1 lepton out" or something like this).

One could make the same remark about the syntax for `BrLTo3L`. In the end it doesn't matter. It's just that it would be nice to have it consistent. I'm fine with both versions.
 
> Another point to consider: Is one of the syntax proposals (old or new) more-or-less consistent with decay specifications? For example one could imagine that in the future we may support certain specific decays with a syntax like `Decay[Fe[2] -> {Fe[2], hh[1]}]` or so. Such a syntax would then be very clear and consistent with the old syntax for `BrLToLGamma`.

I think this situation is different. Because `Decays` don't have names of particles in function name (like `BrLTo3L` where you know it's a lepton). So for decays I'd use `Decay[Fe[2] -> {Fe[2], hh[1]}]` syntax. Actually,you raise a very good point. What I imagine is that at some point we we'll add an option to allow users to add specific 3-body decays using `NPointFunctions`. For example, one needed thing is `Decay[Hp -> {bar[Fd][3], Fd[3], Wp}]`.

Reply:
Ok, then for consistency I'd vote for allowing the `BrLToLGamma[Fe[2] -> {Fe[1], VP}]` syntax at least. It could be a verbose version of a shorter form.